assets/ds-moon-preset.js

```diff
@@ -319,9 +319,19 @@ module.exports = {
         "popo": {
           DEFAULT: withOpacityValue("--popo"),
         },
+        "jiren": {
+          DEFAULT: withOpacityValue("--jiren"),
+        },
+        "heles": {
+          DEFAULT: withOpacityValue("--heles"),
+        },
+        "zeno": {
+          DEFAULT: withOpacityValue("--zeno"),
+        },
         black: "#000",
-        hover: "rgba(34, 34, 37, 0.12)",
-        "primary-hover": "rgba(0, 0, 0, 0.08)",
+        hover: "rgba(0, 0, 0, 0.04)",
```

Comment:
Why rgba here, these should be using hover color tokens. Especially considering light and dark theme have different values for them.

Reply:
Thanks for you comment and input! We will remove these hover declarations from this particular file.